tests/integration/shared/sources/new-or-changed-sources-test.js

```diff
@@ -0,0 +1,195 @@
+const path = require('path')
+const fs = require('fs')
+const test = require('tape')
+
+const srcShared = path.join(process.cwd(), 'src', 'shared')
```

Comment:
These are all much tidier if above we `const { join } = require('path')`

Reply:
Fixed, thanks.  I used to use `path` for something else, since removed.   Cheers!